### ISSUE TITLE
Extracting human36 parsed images

### ISSUE THREAD

**Issue by vigneshrk29** [mks0601/3DMPPE_POSENET_RELEASE#131]
Hello,

I am trying to extract the parsed images of human36m. I could only get images.tar.gzaa to work the sh command. But I get the following error: 
gzip: stdin: unexpected end of file
images/s_05_act_06_subact_01_ca_02/s_05_act_06_subact_01_ca_02_000871.jpg
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now

Which results in the following error when I run test.py:
OSError: Fail to read .[./data/Human36M/images/s_09_act_02_subact_01_ca_01/s_09_act_02_subact_01_ca_01_000001.jpg](https://vscode-remote+wsl-002bubuntu.vscode-resource.vscode-cdn.net/home/vigneshrk/projects/Internship/3DMPPE_POSENET_RELEASE/data/Human36M/images/s_09_act_02_subact_01_ca_01/s_09_act_02_subact_01_ca_01_000001.jpg)

How do I fix this?

**Comment by vigneshrk29** [comment]
Additionally, do I need all the images.tar.gzaa files to run test.py?

**Comment by mks0601** [comment]
Download all tar.gz* and run sh extract_targz.sh

**Comment by vigneshrk29** [comment]
Hi,

I downloaded all the files and ran all the data. Still some folders required for testing are missing. Like s_11_act_06_subact_02_ca_02. Is it possible to just share the parsed data of subjects 9 and 11? Else how to proceed?

Thanks

**Comment by mks0601** [comment]
I think you didn't download properly.
1. Download all tar.gz* and do not change their name
2. check md5sum
3. run the script that I uploaded (extract_targz.sh)

**Comment by vigneshrk29** [comment]
Hi,

I re-downloaded everything, ran md5sum and ran the script. Still the error that some images are missing keeps occuring. Is there:
1 ) a way to specify a specific folder from the downloaded images to run testing
2) Just get the parsed data for subject 11 and 9

**Comment by mks0601** [comment]
I've just downloaded and checked there is no problem. I can see s_11_act_06_subact_02_ca_02.
Are you using Window?
To answer your questions, there is no way for that.

**Comment by vigneshrk29** [comment]
Yes, I am working on windows (WSL). The data is unzipped to an external hardisk as I dont have that much space on my laptop. For example, how many times I run the extract code, the folder s09_act_02_subact_01_01 is never unzipped

**Comment by mks0601** [comment]
I guess that is the problem. I have seen some guys fail to unzip data with Window. Do you have any linux machines?

**Comment by vigneshrk29** [comment]
I have windows subsystem for linux but the space is not sufficient. Is there anyway to make it work on a windows system?

**Comment by mks0601** [comment]
sorry I have no window machine, so I can't do any test..

**Comment by tribeband** [comment]
I run the command in Linux. but it still return wrong.
File "E:\Tools\anaconda\envs\cv\lib\site-packages\torch\utils\data\_utils\collate.py", line 163, in collate_tensor_fn
    return torch.stack(batch, 0, out=out)
RuntimeError: stack expects each tensor to be equal size, but got [15, 3] at entry 0 and [16, 3] at entry 1

The files download saved in the harddisk:
augmented_set.tar-004.gzaa,
augmented_set.tar-007.gzab,
augmented_set.tar-005.gzac,
augmented_set.tar-006.gzad

**Comment by mks0601** [comment]
Please do not change the filenames.
The original ones do not have -004, -007 like this.
And run extract_targz.sh to unzip them
